### ISSUE TITLE
[a11y]: Missing labels 

### ISSUE THREAD

**Issue by CBID2** [codedex-io/css-101#55]
### Description 
In my 2nd review of the HTML file for [the local library exercise](https://github.com/codedex-io/css-101/blob/main/3-box-model/18-local-library/index.html), I noticed that form was missing label elements:

https://github.com/codedex-io/css-101/blob/36f039d1aa3629bc0cf396d7ff77e6e4913ecfe2/3-box-model/18-local-library/index.html#L25-L29

### Solution
Add `<label>` tags and their respective `for` attributes
